styles.css

```diff
@@ -0,0 +1,190 @@
+body {
+  background-color: #191765;
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  width: 100%;
+}
+
+@media only screen and (min-width: 375px) {
+  /* Header Styling */
+  .navbar {
+    display: flex;
+    color: white;
+    position: relative;
+    top: 43px;
+    justify-content: space-around;
+    width: calc(100% - 100px);
+  }
+
+  .header-name {
+    color: white;
+    text-decoration: none;
+    font-size: 18px;
+  }
+
+  .hamburger-menu {
+    position: relative;
+    left: 103px;
+  }
+
+  /* Introduction Styling */
+
+  .introduction-image {
+    background-image: url(images/Ellipse18.svg);
+    margin: 0;
+    margin-left: 0;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 225px;
+    position: absolute;
+  }
+
+  .introduction {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    min-height: 100px;
+  }
+
+  .introduction-name {
+    color: #7d73fc;
+    font-size: 48px;
+    position: relative;
+    text-align: center;
+    margin-top: 94px;
+    left: 46px;
```

Comment:
- [ ] Kindly fix the design as per Figma template. You must have noticed that it's aligned to the right because of which the content cant be seen. Make sure to align in the centre.
- [ ] Kindly fix the background image as per the figma design.
Your design:
<img width="355" alt="Screenshot 2022-03-27 at 1 19 11 AM" src="https://user-images.githubusercontent.com/70269507/160258907-70567fbe-ee1b-4564-b912-e7fdf23dbddb.png">
Figma design:
<img width="330" alt="Screenshot 2022-03-27 at 1 20 24 AM" src="https://user-images.githubusercontent.com/70269507/160258918-c9549b71-d637-4dce-9a13-2053bde913fb.png">